packages/analytics-types/src/config/browser.ts

```diff
@@ -7,17 +7,64 @@ import { PageTrackingOptions } from '../page-view-tracking';
 export interface BrowserConfig extends ExternalBrowserConfig, InternalBrowserConfig {}
 
 export interface ExternalBrowserConfig extends Config {
+  /**
+   * An app version for events tracked. This can be the version of your application.
+   * @defaultValue `undefined`
+   */
   appVersion?: string;
+  /**
+   * The default event tracking configuration.
+   * @defaultValue `true`
+   */
   defaultTracking?: boolean | DefaultTrackingOptions;
+  /**
+   * The identifier for the device running your application.
+   * @defaultValue `UUID()`
+   */
   deviceId?: string;
+  /**
+   * Configuration for cookie.
+   */
   cookieOptions?: CookieOptions;
+  /**
+   * The storage for user identify.
+   * @defaultValue `"cookie"`
+   */
   identityStorage?: IdentityStorageType;
+  /**
+   * The partner identifier.
+   * Amplitude requires the customer who built an event ingestion integration to add the partner identifier to partner_id.
+   * @defaultValue `undefined`
+   */
   partnerId?: string;
+  /**
+   * The custom Session ID for the current session.
+   * @defaultValue `timestamp`
+   */
   sessionId?: number;
+  /**
+   * The period of inactivity from the last tracked event before a session expires in milliseconds.
+   * @defaultValue `1800000`
```

Comment:
can we add delimiter for readability? 
```suggestion
   * @defaultValue `1,800,000` (30 minutes)
```

Reply:
Updated.